UnitedIncome/components
# [APP-4012] CardContentDescription Validation

### Description
> Please provide a brief description of your changes below.

Adds some additional validation to the `CardContentDescription` component so it now invalidates the McG card when the fetch status is in flight. Also added a prop called `isValid` which will essentially hide the component if a criteria isn't met. This last part is needed in some circumstances where we always need to make a fetch request but not show the component.  (See auto loan flow)

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Enter input in the box in Storybook. 
* If the component is fetching the continue button will be greyed out.

### Comments
> Any additional comments that will aid with the review process.

I don't love this solution, I'm open to suggestions if you can think of something better.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4012)
